invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/EItem/EItemForms/EItemEditor/components/EItemForm/EItemForm.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { getIn } from 'formik';
+import { Form } from 'semantic-ui-react';
+import _ from 'lodash';
```

Comment:
Import only the function you will use not the whole lib
i.e. `import pick from 'lodash/pick';`

Reply:
Yes please, I didn't notice this while reviewing, this is rather important